rpcs3/Emu/Cell/Modules/cellGame.h

```diff
@@ -57,19 +57,19 @@ enum
 	CELL_GAME_DIRNAME_SIZE       = 32,
 	CELL_GAME_HDDGAMEPATH_SIZE   = 128,
 	CELL_GAME_THEMEFILENAME_SIZE = 48,
+
 	CELL_GAME_SYSP_LANGUAGE_NUM  = 20,
 	CELL_GAME_SYSP_TITLE_SIZE    = 128,
 	CELL_GAME_SYSP_TITLEID_SIZE  = 10,
 	CELL_GAME_SYSP_VERSION_SIZE  = 6,
 	CELL_GAME_SYSP_PS3_SYSTEM_VER_SIZE = 8,
 	CELL_GAME_SYSP_APP_VER_SIZE  = 6,
 
-	CELL_GAME_GAMETYPE_SYS  = 0,
-	CELL_GAME_GAMETYPE_DISC = 1,
-	CELL_GAME_GAMETYPE_HDD  = 2,
-	CELL_GAME_GAMETYPE_HOME = 4,
-
+	CELL_GAME_GAMETYPE_SYS      = 0,
+	CELL_GAME_GAMETYPE_DISC     = 1,
 	CELL_GAME_GAMETYPE_GAMEDATA = 3,
+	CELL_GAME_GAMETYPE_HDD      = 2,
+	CELL_GAME_GAMETYPE_HOME     = 4,
```

Comment:
if you're going to reorder enums, why go with 0,1,3,2,4? Triggered

Reply:
Ugh good catch